src/pages/admin/profile/[user_id].tsx

```diff
@@ -0,0 +1,128 @@
+import { Suspense, useContext, useEffect, useState } from "react";
+
+import { NextRouter, useRouter } from "next/router";
+import { userContext } from "supabase/user_context";
+import { useSupabaseClient } from "@supabase/auth-helpers-react";
+import { Database } from "supabase/db_types";
+
+import { Avatar, Chip, Typography, Stack, Button } from "@mui/material";
+import MaleIcon from "@mui/icons-material/Male";
+import FemaleIcon from "@mui/icons-material/Female";
+import TransgenderIcon from "@mui/icons-material/Transgender";
+import CakeIcon from "@mui/icons-material/Cake";
+
+import Navbar from "@/components/admin/AdminNavbar";
+import Loading from "@/components/public/Loading";
+
+import { User } from "@/types/User";
+import { PagePaths } from "enum/pages";
+import { Gender } from "enum/gender";
+import { GetUserByUserId } from "@/services/User";
+import AdminVerifyDialog from "@/components/admin/AdminVerifyDialog";
+import VerifyChip from "@/components/profile/VerifyChip";
+
+// style
+const profile_layout = {
+  width: "30vw",
+  minWidth: "200px",
+};
+const avatar = { width: 200, height: 200 };
+
+export default function adminProfile() {
+  const router: NextRouter = useRouter();
+  const userStatus = useContext(userContext);
+  const supabaseClient = useSupabaseClient<Database>();
+
+  const [targetUserData, setTargetUserData] = useState<User | null>(null);
+
+  const [isVerifyModalShow, setIsVerifyModalShow] = useState<boolean>(false);
+
+  useEffect(() => {
+    async function getTargetUserData() {
+      if (!userStatus.user || !router.query.user_id || targetUserData) return;
+      const userData = await GetUserByUserId(router.query.user_id as string, supabaseClient);
+      setTargetUserData(userData);
+    }
+
+    getTargetUserData();
+  }, [router.query.user_id, supabaseClient, userStatus.user, targetUserData]);
+
+  function openVerifyModal(): void {
+    setIsVerifyModalShow(true);
+  }
+  function closeVerifyModal(): void {
+    setIsVerifyModalShow(false);
+  }
+
+  if (userStatus.isLoading) return <Loading />;
+  if (!userStatus.user) {
+    router.push(PagePaths.login);
+    return;
+  }
+  if (!userStatus.user.isAdmin) {
+    router.push(PagePaths.home);
+    return;
+  }
```

Comment:
right now,  admin coould verify himself or other admin. Maybe route them to somewhere else?

Reply:
Now we don't have admin profile anymore...